sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -511,7 +552,29 @@ object ScalaJSPlugin extends AutoPlugin {
     Seq(
         scalaJSStage := Stage.FastOpt,
         scalaJSUseRhinoInternal := false,
-        ScalaJSPluginInternal.scalaJSClearCacheStats := globalIRCache.clearStats()
+        ScalaJSPluginInternal.scalaJSClearCacheStatsInternal := {},
+
+        // Clear the IR cache stats every time a sequence of tasks ends
+        onComplete := {
+          val prev = onComplete.value
+
+          { () =>
+            prev()
+            globalIRCache.clearStats()
```

Comment:
We should also report the stats here, rather than each time we are kind of done with the cache. This will also allow us to make the `logIRCacheStats` private.

Reply:
I tried, but here we only have access to `sLog` (instead of `streams`) because we're in a setting. And things sent to the `sLog` are (apparently) not displayed by `last`. So any `sLog.value.debug(...)` is lost and can never be observed. (a `.info(...)` is printed on the console, but not stored for further redisplay by `last` either)

Reply:
Ugh :( I was really hoping we could get rid of the ugly "invoke this whenever you have done something that has used the cache" :(
Let's leave it then for now.